### PR TITLE
feat(file): Added download attributes. - OEL-2772

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Comment by julien-** [comment]
@tibi2303 Did not you forget to remove [download forced to true](https://github.com/openeuropa/bootstrap-component-library/pull/573/files#diff-eb810e757cee54f8941c802d1a48f18c2f72c997072bbe05553ec56be6ce9ae7R154) ? ?

**Comment by julien-** [comment]
Please check https://github.com/openeuropa/bootstrap-component-library/pull/583/files

**Comment by julien-** [comment]
See #583

File: src/compositions/bcl-file/file.test.js [hunk 1]
```diff
@@ -87,6 +87,23 @@ describe("OE - File Card", () => {
...
+    expect.assertions(1);
+    return expect(
+      render({
+        ...demoCardData,
+        download: {
+          label: "Download",
+          path: "/example.html",
+          icon: {
+            name: "cloud-download",
+            size: "fluid",
```
[the remark this answers is not in the document]
Comment (tibi2303) [reply src/compositions/bcl-file/file.test.js:100]: In the test you can see that it's not populated... I could add but we have only that sprite :)) and i think it's enough that it's not populated if you dont add it (that means you can change it how you want).

File: src/compositions/bcl-file/file.html.twig [hunk 2]
```diff
@@ -138,14 +138,27 @@
...
-          icon: {
-            name: "download",
-            size: "fluid",
-            path: _icon_path,
-          },
-          attributes: create_attribute()
+          attributes: _download.attributes
                       .setAttribute('download', true)
                       .setAttribute('target', '_blank')
                       .addClass(['standalone', 'align-self-center', 'd-inline-block', 'mt-1', 'mt-md-0', 'flex-shrink-0'])
```
[the remark this answers is not in the document]
Comment (tibi2303) [reply src/compositions/bcl-file/file.html.twig:154]: It was failing like this, it's more or less how you suggested now but without the attributes merge.

**Pull request merged**